frontend/src/main/scala/akka/viz/frontend/ApiConnection.scala

```diff
@@ -1,44 +1,47 @@
 package akka.viz.frontend
 
-import org.scalajs.dom._
-import org.scalajs.dom.raw.WebSocket
+import org.scalajs.dom
+import org.scalajs.dom.raw._
 
-object ApiConnection {
+import scala.concurrent.Promise
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/lustefaniak/akka-viz/file/2051092138/issues/source?bid=3121282&fileBranchId=3133751#l6)